exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillFactory.java

```diff
@@ -37,6 +38,24 @@
   protected final int major;
   protected final int minor;
 
+  static {
+    Properties prop = new Properties();
```

Comment:
Actually, I think this code can be moved into `SecurityConfiguration.java`.

Java handles resources by looking in the class path, which is formed from all jars. A long as the resource file is somewhere in the top level of some jar, this code will find it. So, despite what I said in person the other day, the code need not be here to work.

That said, the property file DOES have to be in the JDBC-all package to avoid having multiple files of the same name in the class path.

Please try it out to see if the code works in `SecurityConfiguration.java`.